tests/e2e/lib/subscription_helpers.go

```diff
@@ -39,6 +30,33 @@ func (d *DpaCustomResource) GetOperatorSubscription(c client.Client, stream stri
 	return &Subscription{&sl.Items[0]}, nil
 }
 
+func (d *DpaCustomResource) GetOperatorSubscription(c client.Client, stream string) (*Subscription, error) {
+	err := d.SetClient(c)
+	if err != nil {
+		return nil, err
+	}
+
+	label := ""
+	if stream == "up" {
+		label = "operators.coreos.com/oadp-operator." + d.Namespace
+	}
+	if stream == "down" {
+		label = "operators.coreos.com/redhat-oadp-operator." + d.Namespace
+	}
+	return GetOperatorSubscription(c, d.Namespace, label)
+}
+
+func (v *VirtOperator) GetOperatorSubscription(stream string) (*Subscription, error) {
+	label := ""
```

Comment:
are both streams necessary? are we interested in testing both?
![Screenshot from 2023-12-28 14-20-35](https://github.com/openshift/oadp-operator/assets/66965232/cff6bbf4-acb4-42a2-9773-80703f2b64bf)

Reply:
I don't really know, I just saw that there was existing subscription-related code that did the same thing and copied it. I took this out to make it simpler for now.